src/main/java/org/folio/qm/controller/RecordsEditorRecordsApi.java

```diff
@@ -29,8 +32,9 @@
   }
 
   @Override
-  public ResponseEntity<Void> recordsPost(@Valid QuickMarc quickMarc) {
-    return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+  public ResponseEntity<CreationStatus> recordsPost(@Valid QuickMarc quickMarc) {
+    CreationStatus status = marcRecordsService.createNewInstance(quickMarc, folioExecutionContext);
```

Comment:
I think that `FolioExecutionContext` bean could be injected to the service (where it really needed) instead of the controller.

Reply:
thanks for note, moved